### ISSUE TITLE
sync from windows local bug

### ISSUE THREAD

**Issue by tlightsky** [NebulousLabs/Sia-Nextcloud#4]
I paster 4 files to sia external storage folder in my windows nextcloud folder
2 of them upload successfully
other 2 is not

They keep upload progress 0%,
I check the `./renter/renter.json` file, 
the RepairPath which is `/tmp/oc_tmp_XSiItC` and `/tmp/oc_tmp_o5xRdC` both no files there.

I think this could be a problem

**Comment by tlightsky** [comment]
try to upload from web,
```
                "IMG_2777.jpg.ocTransferId1630880273.part": {
                        "RepairPath": "/tmp/oc_tmp_vfiolY"
                },
```
file `/tmp/oc_tmp_vfiolY` file also deleted which makes sia upload progress keeps 0%

**Comment by avahowell** [comment]
I've pushed a fix for this in #4. Because of the way the renter currently works, you'll have to provide a second configuration option, `datadir`, which is the directory to keep the files in on the nextcloud server so the renter can continuously repair them.

**Comment by tlightsky** [comment]
@johnathanhowell thanks for reply,
yes,but that also takes space on server, which might makes external storage meaningless.
`Ability to repair files without having the file locally` might help, but still waiting...
https://trello.com/c/VE7DJMZN/38-ability-to-repair-files-without-having-the-file-locally
